README.md

```diff
@@ -3,6 +3,33 @@
 [![Backend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Backend%20CI/badge.svg)](https://github.com/aggie-coding-club/rev-registration/actions?query=branch%3Amaster+workflow%3A%22Backend+CI%22)
 [![Frontend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Frontend%20CI/badge.svg)](https://github.com/aggie-coding-club/rev-registration/actions?query=branch%3Amaster+workflow%3A%22Frontend+CI%22)
 
+![project-demo](assets/project-demo.gif)
+
+## Description
+
+Rev Registration is an automatic class scheduler for Texas A&M, aimed at easing the process
+of course registration by  generating compatible schedules given the courses a student would
+like to take and what preferences they have. Simply select a term, pick your courses, mark off when
+you're not available, and we'll generate schedules for you!
+
+For instance, imagine you've settled on 3 sections of a course you are fine with having and are having
+trouble finding a schedule without conflicts. Select all 3 under the Section customization level
+and let the schedule generator create schedules for you without time conflicts!
+
+**Features:**
+
+- Automatic schedule generation
+- Blocking off busy times (availabilities)
+- Grade distribution displaying
+- Displaying current seats remaining
```

Comment:
Quite nitpicky but as these are pretty similar statements, it would probably read a bit smoother with some parallel structure. You could do:
"Displays grade distributions
 Displays current seats reaming".
If that seems too monotone, I would keep the verb at the front as most of the bullets already follow that pattern but use a similar word for one of the points kinda like this:
"Displays grade distributions
 Shows current seats reaming".

Reply:
Changed to use the "unconjugated verbs", as I do agree with you + Ryan that it sounds better (I imagined saying "you can X" in front of the features, although it doesn't work with "Automatic schedule generation" nor "visual display of schedules" lol)